kernel/kernel-api/src/main/java/io/delta/kernel/coordinatedcommits/CommitCoordinatorUtils.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (2024) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.delta.kernel.coordinatedcommits;
```

Comment:
Is this file copied from some other file?

Reply:
Great question / callout. Yes, these methods are taken from https://github.com/delta-io/delta/blob/master/storage/src/main/java/io/delta/storage/commit/CoordinatedCommitsUtils.java

We will be deleting that file when this refactor is done

Reply:
I'm assuming you will move other methods from that file here as well eventually.